app/app.go

```diff
@@ -138,6 +143,11 @@ var (
 			authzmodule.AppModuleBasic{},
 			ecocreditmodule.Module{},
 			data.Module{},
+			gov.NewAppModuleBasic(
```

Comment:
moved this out of `customModuleBasics()`, since its an sdk module, not really a custom module

Reply:
I think this should be in `stable_appconfig` because we have two sets of gov handlers.
one with `cosmwasm` in experimental config
and another one without `cosmwasm` in stable config.